src/olympia/accounts/views.py

```diff
@@ -284,7 +298,7 @@ def post(self, request, user, identity, next_path):
         else:
             update_user(user, identity)
             response = Response({'email': identity['email']})
-            add_api_token_to_response(response, user, set_cookie=False)
+            add_api_token_to_response(response, user)
```

Comment:
Yes, I did that for consistency, there was no point in not doing it in that view (which is /login/ instead of /authenticate/ - I'm not sure if it's still used ?)